ArduCopter/mode_acro.cpp

```diff
@@ -51,14 +51,41 @@ void ModeAcro::run()
     }
 
     // run attitude controller
-    attitude_control->input_rate_bf_roll_pitch_yaw(target_roll, target_pitch, target_yaw);
+    if (g2.acro_options.get() & uint8_t(AcroOptions::RATE_LOOP_ONLY)) {
+        attitude_control->input_rate_bf_roll_pitch_yaw_2(target_roll, target_pitch, target_yaw);
+    } else {
+        attitude_control->input_rate_bf_roll_pitch_yaw(target_roll, target_pitch, target_yaw);
+    }
 
     // output pilot's throttle without angle boost
     attitude_control->set_throttle_out(get_pilot_desired_throttle(),
                                        false,
                                        copter.g.throttle_filt);
 }
 
+bool ModeAcro::init(bool ignore_checks)
+{
+    if (g2.acro_options.get() & uint8_t(AcroOptions::AIR_MODE)) {
+        disable_air_mode_reset = false;
```

Comment:
I probably would have moved this outside the "if" so that it's always reset during the init.  It doesn't make any functional difference though.